components/ResilienceResource/MindfulnessExercises.js

```diff
@@ -5,8 +5,12 @@ import ColourBoxContainer from './ColourBoxContainer'
 import styles from './MindfulnessExercises.module.css'
 
 function MindfulnessExercises({ exercises }) {
+  let classStyles = []
+  classStyles.push(styles.MindfulnessExercises)
+  classStyles.push('container')
+  classStyles = classStyles.join(' ')
```

Comment:
lol what's the purpose of push class styles when you can just initialize them in the array?

Reply:
yea I find that annoying too. It's cuz they're different types.. you have to combine objects and string literals